index.html

```diff
@@ -244,6 +244,20 @@ <h1>I'm Naan.
                     </div>
                 </div>
             </div>
+            <section class="contact_me">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+                <form action="https://formspree.io/f/xvonabyg"
+                method="POST">
+                    <ul>
+                        <li><input value="" placeholder="Enter your name" id="name" type="text" name="user_name" maxlength="30" required></li>
+                        <li><input value="" placeholder="Enter your email" id="email" type="email" name="email" required></li>
+                    </ul>
+                    <textarea value="" id="message" placeholder="Write your message here" type="text" name="message" maxlength="500" required></textarea>
+                    <button type="submit">Get in touch</button>
+                </form>
+            </section>
         </section>
         
```

Comment:
- [ ] Please, kindly fix the `overflow-x` anomaly, to enable you to have a very beautiful design,

![mobile](https://user-images.githubusercontent.com/41163571/220789414-4d21e2fd-3a8f-41d4-a9fd-01980f178f06.gif)

Reply:
Thank you for review @AdnanOlarmmi. Which section should I fix the overflow anomaly on and what property of overflow should I use?

Reply:
Hey @naanmohammed you are using `overflow-x: hidden;` property on the body which is the right property to use however I can see in your code you are using absolute widths like `px` on your section containers and content (`<p>,<h1>, etc`) which is making your content overflow-x. Please use relative values like % for the width of your elements and containers.

Reply:
Hello @shella12 thank you for the comment. I am making the changes. I will tag you when done.